### ISSUE TITLE
[Question] Kitty detect fonts name is not consistent

### ISSUE THREAD

**Issue by haunt98** [d8-bbc11/monolisa#13]
Hi, thanks for your wonderful effort.

Just having some hiccups with kitty

<img width="832" alt="image" src="https://github.com/d8-bbc11/monolisa/assets/5424272/1a8bfaff-9dd2-44e8-b21d-7a568d8da713">

As you see there are 2 font names not consistent with others

```
MonoLisa Italic (Italic)
MonoLisa MonoLisa (MonoLisa)
```

**Comment by d8-bbc11** [comment]
aside from regular being called 'MonoLisa', i don't see the exact issue here

should it be something like "MonoLisa Regular" and "MonoLisa Regular Italic" instead?

**Comment by haunt98** [comment]
Yeah, it's not issue, I can use the font easily. Just curious about the inconsistent.

For example Victor mono fonts show:

```txt
Victor Mono Italic (VictorMono-Italic)
Victor Mono Regular (VictorMono-Regular)
```

<img width="1352" alt="image" src="https://github.com/d8-bbc11/monolisa/assets/5424272/d354de99-df7a-4a2f-bd63-9b0e22c9ab97">

**Comment by d8-bbc11** [comment]
it's caused by my crack build/publishing script, i just forgot to change those weight names in it

gonna fix soon
